.env.example

```diff
@@ -96,6 +97,11 @@ MEDIUM_GROQ_MODEL=     # Default: llama-3.3-70b-versatile
 LARGE_GROQ_MODEL=      # Default: llama-3.2-90b-vision-preview
 EMBEDDING_GROQ_MODEL=  # Default: llama-3.1-8b-instant
 
+# NanoGPT Configuration
```

Comment:
```suggestion
# NanoGPT Configuration
NANOGPT_API_KEY= # NanoGPT API Key
```

@odilitime do you think we should group the NanoGPT information together? More of a nitpick, but this will help devs with readability and configuration.

Reply:
When I run this, I see this error. @dylan1951 do you know why this happens?
![image](https://github.com/user-attachments/assets/e4224393-416c-47ae-ad90-5b6e055747d6)